stock_picking_batch_oca/models/stock_batch_picking.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+
+from odoo.exceptions import UserError
+
+
+class StockBatchPicking(models.Model):
+    """ This object allow to manage multiple stock.picking at the same time.
+    """
+    # renamed stock.batch.picking -> stock.picking.batch
+    _inherit = 'stock.picking.batch'
+
+    name = fields.Char(
+        string='Name',
+        index=True,
+        unique=True,
+        states={'draft': [('readonly', False)]},
+        default=lambda self: self.env['ir.sequence'].next_by_code(
+            'stock.picking.batch'
+        ),
+    )
+
+    # added state to be compatible with picking_ids
+    state = fields.Selection(
+        selection_add=[
+            ('assigned', 'Available'),
+        ],
+        string='State',
+        readonly=True, index=True,
+        help='the state of the batch picking. '
+             'Workflow is draft -> in_progress -> done or cancel',
```

Comment:
what about 'assigned/Available'?

Reply:
in progress = assigned 
one used for odoo, other for oca, doc updated